### PR TITLE
docs: default to blocks for getting started

### PR DESCRIPTION
**Pull request by jtlisi**
**What this PR does**:

Currently, are the getting started page of the Cortex documentation presents new users with three choices:

- Blocks
- Chunks
- Gossip

I believe immediately confronting new users looking to demo the project with these choices is a mistake. In this PR I chose to default to using blocks storage for the getting started guide. I have moved the other guides under the `guides` section of the documentation.

### CODE REVIEW

File: docs/guides/chunks-storage-getting-started.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Getting Started with Chunks Storage"
```
Comment (pracucci) [inline docs/guides/chunks-storage-getting-started.md:2]: Because of this change, could you fix the link in `docs/case-studies/buoyant-cloud.md` please?

Comment (pracucci) [reply docs/guides/chunks-storage-getting-started.md:2]: Also, what if we do a further step and move this doc under `docs/chunks-storage/`?

**Review by pracucci — APPROVED** [review]
LGTM (modulo a couple of nits)

File: docs/guides/chunks-storage-getting-started.md (hunk 1 above)
Comment (jtlisi) [reply docs/guides/chunks-storage-getting-started.md:2]: I think that makes sense. @pracucci WDYT about also moving the `docs/guides/running-chunks-storage-in-production.md` and ` docs/guides/running-chunks-storage-with-cassandra.md` under that section as well?

Comment (pracucci) [reply docs/guides/chunks-storage-getting-started.md:2]: I think it makes sense. We're heading towards deprecating the chunks storage (even if very soon) so I agree having the blocks storage doc to be more prominent.

**Review by pracucci — APPROVED** [review]
Good job, thanks!

**Pull request merged**